packages/modules/web-viewer-react/package.json

```diff
@@ -73,8 +73,8 @@
     "@itwin/core-frontend": "^4.0.0",
     "@itwin/core-react": "^4.0.0",
     "@itwin/presentation-common": "^4.0.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
update the peers for the modules as per the other comment

Reply:
updated peers